src/index.js

```diff
@@ -403,11 +403,13 @@ AFRAME.registerComponent('editor', {
 
 	AFRAME.registerComponent('n-sound', {
 		init: nativeComponentInit,
-		pause: function () {
+		pauseSound: function () {
```

Comment:
Seems like we should hook into it naturally since it something we would get for free. This change doesn't preclude us from doing so. Although, I'm not sure if the a-frame's play/pause system is really that well established because even they suggest using methods directly on the sound component.